test/scr_parse_test.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+RSpec.describe "slp.conf parsing" do
+  COMMENTED_LINES = File.join(FIXTURES_PATH, "commented-slp.conf")
+  UNCOMMENTED_LINES = File.join(FIXTURES_PATH, "uncommented-slp.conf")
+
+  COMMENTED_REGEXP = Regexp.new '^[#;][ \t]*([^ \t\=]+)[ \t\=]*[ ]*(.+)[ \t]*$'
+  UNCOMMENTED_REGEXP = Regexp.new '[ \t]*([^ \t\=]+)[ \t\=]+[ ]*(.+)[ \t]*$'
```

Comment:
Aw, this duplication of the regexp is quite error prone I fear. Not to mention subtle differences between C and Ruby regexes. I think I can make a test that uses the production parser.